Source/WebKit/UIProcess/Extensions/WebExtensionTab.h

```diff
@@ -65,15 +65,21 @@ class WebExtensionTab : public RefCounted<WebExtensionTab> {
         Title      = 1 << 7,
         URL        = 1 << 8,
         ZoomFactor = 1 << 9,
-        All        = std::numeric_limits<uint16_t>::max(),
+        All        = Audible | Loading | Muted | Pinned | ReaderMode | Size | Title | URL | ZoomFactor,
```

Comment:
Was this part of a different PR too?

Reply:
Yeah I moved it here, it was in the other one by mistake.